src/patterns/news/index.njk

```diff
@@ -0,0 +1,87 @@
+---
+title: News
+group: Pages
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+{% from "components/external-link/_macro.njk" import onsExternalLink %}
+{% from "components/lists/_macro.njk" import onsList %}
+
+How to display, tag and categorise news articles.
+
+{{
+    patternlibExample({ "path": "patterns/news/examples/landing/index.njk" })
+}}
+
+## When to use this pattern
+
+Use this pattern to publish a news article to a public-facing website.
+
+## How to use this pattern
+
+Use the [articles component](/components/articles) to build consistent articles.
+
+### News index
```

Comment:
Now that we're keeping this heading, I would move the main example down under the 'news index' content.